torch/_torch_docs.py

```diff
@@ -1358,6 +1359,7 @@ def merge_dicts(*dicts):
     input (Tensor): the tensor to be added
     batch1 (Tensor): the first batch of matrices to be multiplied
     batch2 (Tensor): the second batch of matrices to be multiplied
+    out_dtype (dtype, optional): the dtype of the output tensor. Supported combination is torch.float32 for torch.float16/torch.bfloat16 input
```

Comment:
also i'm not actually sure if the CI lint will catch this, but it looks like for longer lines we move them to a newline and indent in a certain way. Probably worth trying to emulate for these longer lines too? example: https://github.com/pytorch/pytorch/blob/main/torch/_torch_docs.py#L2318